src/xhr.js

```diff
@@ -14,7 +14,7 @@ function ConnectionError(message) {
   this.stack = (new Error()).stack;
 }
 
-ConnectionError.prototype = Object.create(Error.prototype);
+ConnectionError.prototype = new Error();
```

Comment:
Looks like this thread explains some reasons why we shouldn't do this: http://stackoverflow.com/questions/783818/how-do-i-create-a-custom-error-in-javascript